lib/core/src/control/event_loop.rs

```diff
@@ -95,21 +103,23 @@ impl Default for EventLoopData {
 
 impl EventLoopData {
     /// Create new instance.
-    pub fn run(&mut self) {
+    pub fn run(&mut self, time_ms:f32) {
         (self.on_loop_started)();
         let callbacks   = &mut self.callbacks;
         let callback_id = self.main.as_ref().map_or(default(), |main| {
-            callbacks.run_all();
+            callbacks.run_all(time_ms);
```

Comment:
I have to admit that the naming of variables in this and previous PR is so nice. It is **MUCH** better than it was in the first PRs. Really good job Danilo! ❤️

Reply:
Thx :D